db/migrate/20220823044927_add_payment_pointer.rb

```diff
@@ -0,0 +1,6 @@
+class AddPaymentPointer < ActiveRecord::Migration
+  def change
+    add_column :podcasts, :payment_pointer, :string
```

Comment:
This has changed a bit since this PR was opened, but I believe the new pattern in Feeder is:

We _only_ add the column to the `:feeds` table, unless there's a compelling reason to also add it to `:podcasts`.  And then in the show rss builder do something like:

```ruby
payment_pointer = @feed.payment_pointer || @podcast.default_feed&.payment_pointer
```